ewkcoffea/modules/corrections.py

```diff
@@ -566,3 +567,34 @@ def ApplyJetVetoMaps(jets,year):
     #Sum the outputs for each event (if the sum is >0, the event will fail)
     veto_map_event = ak.sum(jet_vetomap_score, axis=-1)
     return veto_map_event
+
+def CorrectedMETFactory(jets,year,met,syst,isdata):
```

Comment:
Just for my understanding, is this something you would eventually plan to move to tc, or is it too analysis specific?